workspaces/liferay-marketplace-workspace/client-extensions/liferay-marketplace-custom-element/src/manage-app-state/AppManageState.tsx

```diff
@@ -51,6 +51,10 @@ export type InitialStateProps = {
 	priceModel: Specification;
 	productOptionId: number;
 	publisherWebsiteURL: Specification;
+	resourceRequirements: {
+		cpu: number;
+		ram: number;
```

Comment:
I think the input by default is string not number, can you check? there's no problem to use this as string 

it could be:

```suggestion
		cpu: number;
		ram: number;
```

```suggestion
		cpu: number | string;
		ram: number | string;
```

```suggestion
		cpu: string;
		ram: string;
```

I'm saying that in favor of the comment about the initialValue when I suggested to put '' as default value